static/locales/en-US.yaml

```diff
@@ -1007,16 +1011,19 @@ Up Next: Up Next
 #Tooltips
 Tooltips:
   General Settings:
-    Preferred API Backend: Choose the backend that FreeTube uses to obtain data. The
+    Preferred API Backend: "Choose the backend that FreeTube uses to obtain data. The
       local API is a built-in extractor. The Invidious API requires an Invidious server
-      to connect to.
+      to connect to. The Piped API requires a Piped server to connect to and FreeTube currently only supports: playlists, comments and trending from Piped."
```

Comment:
As we are presumably going to add support for Piped in other parts of the app not long after this pull requests, it might make sense to not specify an exact list of supported functionality, so that the translators don't have to retranslate the string for each pull requests.